packages/core/src/browser/style/drag-and-drop.css

```diff
@@ -0,0 +1,14 @@
+#draggable {
```

Comment:
copyright

Reply:
OK, Added the Copyright.
I noticed there is only one other .css file having a Copyright, if we put  it in some .css files, they should all have the copyright

Reply:
I also remove from the css file the section about the draggable, not used yet in this PR.